src/main/java/cl/transbank/util/HttpUtilImpl.java

```diff
@@ -188,13 +189,15 @@ private String formatUrl(String url, String query) {
         return String.format("%s%s%s", url, separator, query);
     }
 
-    private static String getResponseBody(InputStream responseStream) throws IOException {
+    private static String getResponseBody(InputStream responseStream) {
         try (final Scanner scanner = new Scanner(responseStream, StandardCharsets.UTF_8.name())) {
             final String responseBody = scanner.useDelimiter("\\A").next();
             responseStream.close();
 
             logger.log(Level.FINE, String.format("HTTP Response Body : %s", responseBody));
```

Comment:
@goncafa @a1b4 Be careful with logging sensitive information (I wouldn't expect to see the oneclick tokens being logged and I think this line might do it if by accident — or because I'm debugging another problem — I set the production log level to FINE or higher).